src/library/basetracktablemodel.cpp

```diff
@@ -121,7 +122,7 @@ void BaseTrackTableModel::setApplyPlayedTrackColor(bool apply) {
     s_bApplyPlayedTrackColor = apply;
 }
 
-//static
+// static
```

Comment:
No real issue here in this case. In general we try to avoid reformating untouched code to keep the change set minimal. Make sure your IDE does not automatically reformat whole files.

Reply:
Thank you for noting that, I've now set it to only format changes.